functions/src/sendJoTokens.ts

```diff
@@ -0,0 +1,86 @@
+import * as functions from 'firebase-functions';
+import * as util from './util'
+const fs = require('fs')
+const Tx = require('ethereumjs-tx')
+const ethutil = require('ethereumjs-util') 
+
+// Sends Jo tokens from user wallet to another address
+// This function will prepare message and sign it with user(sender) private key
+// then message will be send to JOToken contract by relayer. Relayer will
+// pay the transaction fee.
+// data - {"to": "recipient_address", "value": sent_amount_of_wei}
+export const handler = (data, context, db, web3) => {
+    if(!context.auth) {
+        throw new functions.https.HttpsError('failed-precondition', 'Signed out user call')
+    }
+
+    if(!web3.utils.isAddress(data.to)) {
+        throw new functions.https.HttpsError('invalid-argument', 'Invalid recipient address')
+    }
+
+    if(!(typeof data.value === 'number') || data.value <= 0) {
+       throw new functions.https.HttpsError('invalid-argument', 'Sent value shall be positive number') 
+    }
+
+    // read env variables
+    const jotokenAddress = process.env.JOTOKEN_ADDRESS
+    const relayer = process.env.RELAYER_ADDRESS
+    const relayerPrivKey = process.env.RELAYER_PRIVATE_KEY
+
+    // create JOToken instance
+    const source = fs.readFileSync(require.resolve('./../build/contracts/JOToken.json'))
+    const parsedSource = JSON.parse(source)
+    const JOToken = new web3.eth.Contract(parsedSource.abi, jotokenAddress)
+
+    let senderPrivKey, senderNonce
+
+    // get user(sender) private key and nonce
+    // increase nonce
+    return db.doc(`users/${context.auth.uid}`).get()
+    .then(snapshot => {
+        senderPrivKey = snapshot.data().privateKey
+        senderNonce = snapshot.data().nonce + 1
+        return snapshot.ref.update({nonce: senderNonce})
+    })
+    .then(() => {
+        return web3.eth.getTransactionCount(relayer)
```

Comment:
Same as comment above - in general it's less lines and still  readable when instead of returning one line you just use `() => web3.eth.getTransactionCount(relayer)`. And in this case also maybe just `web3.eth.getTransactionCount(relayer)` will work, without arrow function